### PR TITLE
refactor(remove-storage): remove dead code

### PR DESCRIPTION
**Pull request by AkkarinForest**
This was all dead code (I think :D ). 
I tested quite extensively and everything seems to be working the same.

### CODE REVIEW

File: app/javascript/angular/code/services/_sensors.js [hunk 1]
```diff
@@ -83,24 +83,6 @@ export const sensors = (params, storage, heat, $http) => {
...
-      const callback = data => {
-        storage.updateDefaults({ heat: heat.parse(data) });
-        params.update({ data: { heat: heat.parse(data) } });
-      };
-      $http
-        .get("/api/thresholds/" + sensor.sensor_name, {
-          params: { unit_symbol: sensor.unit_symbol },
-          cache: true
-        })
-        .success(callback);
```
Comment (AkkarinForest) [inline app/javascript/angular/code/services/_sensors.js]: This was used to store heat levels in storage, but that info was never used as far as I can see, so this fetch must be useless.

File: app/javascript/angular/code/services/single_fixed_session.js [hunk 2]
```diff
@@ -22,13 +21,6 @@ angular.module("aircasting").factory("singleFixedSession", [
       selectedStream: function() {
         return this.get().streams[sensors.anySelected().sensor_name];
       },
-      updateHeat: function() {
```
Comment (AkkarinForest) [inline app/javascript/angular/code/services/single_fixed_session.js]: dead function

File: app/javascript/angular/code/services/single_mobile_session.js [hunk 3]
```diff
@@ -16,14 +15,6 @@ angular.module("aircasting").factory("singleMobileSession", [
         return _(mobileSessions.allSelected()).first();
       },
 
-      updateHeat: function() {
```
Comment (AkkarinForest) [inline app/javascript/angular/code/services/single_mobile_session.js]: dead function

**Comment by 3v0k4** [comment]
Good job! And I see you've crossed out from the list we have on the wall. Thanks!

**Pull request merged**